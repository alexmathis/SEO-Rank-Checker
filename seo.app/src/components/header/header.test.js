import React from "react";
import Header from "./header.component";
import { shallow } from "enzyme";

it("expect to render the card component", () => {
  expect(shallow(<Header />)).toMatchSnapshot();
});
