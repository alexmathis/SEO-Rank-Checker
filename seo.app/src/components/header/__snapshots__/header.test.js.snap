// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render the card component 1`] = `ShallowWrapper {}`;
